src/pages/landing/index.js

```diff
@@ -17,21 +17,21 @@ const simple_step_content = [
         text: (
             <Localize translate_text="Open a demo account and start trading for free. Practise with an unlimited amount of virtual funds." />
         ),
-        icon: <img src={PractiseIcon} alt="aim" width="32" height="32" />,
+        icon: <img src={PractiseIcon} width="32" height="32" />,
```

Comment:
We need empty alt tags for these. We can't remove it entirely.

Reply:
I agree on this.. https://web.dev/image-alt/ for reference

Reply:
👌

Reply:
done